packages/ketcher-react/src/script/ui/dialog/toolbox/enhancedStereo/enhancedStereo.tsx

```diff
@@ -53,6 +53,8 @@ const EnhancedStereo: FC<Props> = (props) => {
   const { struct, formState, init, ...rest } = props
   const { result, valid } = formState
 
+  console.log(formState)
```

Comment:
leftover?

Reply:
Oops. I was sure I removed these. Should've checked better. Thanks!
Fixed.